File: backend/src/main/java/org/multimes/backend/core/web/service/interfaces/IMessageService.java
package org.multimes.backend.core.web.service.interfaces;

import org.multimes.backend.core.web.model.dto.requests.SendMessageRequest;
import org.multimes.backend.core.web.model.dto.responses.AllMessagesByDialogIdResponse;

import java.util.List;

public interface IMessageService {
    public void add(SendMessageRequest message);

    List<AllMessagesByDialogIdResponse> getAllByInterId(int id);
}

File: backend/src/main/kotlin/org/multimes/backend/core/web/service/interfaces/IMessageService.kt
package org.multimes.backend.core.web.service.interfaces

import org.multimes.backend.core.web.model.dto.requests.SendMessageRequest
import org.multimes.backend.core.web.model.dto.responses.AllMessagesByDialogIdResponse

interface IMessageService {
    fun add(message: SendMessageRequest)
    fun getAllByInterId(id: Int): List<AllMessagesByDialogIdResponse>
}
